src/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/CertificateRequestChainTests.cs

```diff
@@ -201,6 +218,85 @@ public static void ChainCertRequirements(bool useIntermed, bool? isCA, X509KeyUs
             }
         }
 
+        [Fact]
+        public static void BuildChainUntrustedRoot()
```

Comment:
This test seems to be testing that adding the intermediate to the CustomTrustStore doesn't provide trust.  That's a good test, but the name doesn't really convey that.  `CustomRootTrustDoesNotTrustIntermediates` seems more appropriate (once this test is in DynamicChainTests)